### PR TITLE
Sphinx integration

### PR DESCRIPTION
**Pull request by AStox**
This PR integrates the sphinx cross chain deployer: https://github.com/sphinx-labs/sphinx

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,39 +8,115 @@
...
+
 ```sh
 forge test
 ```
 
+## Deploying
+
+#### Setup
+
+Liquidity Pools uses [Sphinx](https://sphinx.dev/) for crosschain deployments. To deploy the contracts:
```
Comment (hieronx) [inline README.md:48]:
```suggestion
Liquidity Pools uses [Sphinx](https://sphinx.dev/) for cross-chain deployments. To deploy the contracts:
```

File: README.md [hunk 2]
```diff
@@ -7,39 +8,115 @@
...
+and fill in the required environment variables. Then run:
+
+```sh
+yarn install
+yarn sphinx install
+```
+
+#### Configure
+
+Cconfigure the deployment in script/Deployer.sol:
```
Comment (hieronx) [inline README.md:63]:
```suggestion
Configure the deployment in `script/Deployer.sol`:
```

File: README.md [hunk 3]
```diff
@@ -7,39 +8,115 @@
...
+ethereum-mainnet = "https://eth.api.onfinality.io/rpc?apikey=${ONFINALITY_API_KEY}"
+arbitrum-mainnet = "https://arbitrum.api.onfinality.io/rpc?apikey=${ONFINALITY_API_KEY}"
+celo-mainnet = "https://celo.api.onfinality.io/rpc?apikey=${ONFINALITY_API_KEY}"
+base-mainnet = "https://base.api.onfinality.io/rpc?apikey=${ONFINALITY_API_KEY}"
+ethereum-testnet = "https://eth-sepolia.api.onfinality.io/rpc?apikey=${ONFINALITY_API_KEY}"
+```
+
+#### Propose
+
+Sphinx deploys by first proposing deployments, which then need to be approved by a gnosis safe member before they're executed onchain. The safe is automatically generated and assigned to you on account creation at [sphinx.dev](https://sphinx.dev/).
```
Comment (hieronx) [inline README.md:89]:
```suggestion
Sphinx deploys by first proposing deployments, which then need to be approved by a Gnosis Safe member before they're executed onchain. The Safe is automatically generated and assigned on account creation at [sphinx.dev](https://sphinx.dev/).
```

File: README.md [hunk 4]
```diff
@@ -7,39 +8,115 @@
...
+| --------- | ---------------------------------------------------------------------------------------------------------------------------------------------- |
+| Code4rena | [`September 2023 - Code4rena Report`](https://code4rena.com/reports/2023-09-centrifuge)                                                        |
+| SRLabs    | [`September 2023 - SRLabs Report`](https://github.com/centrifuge/liquidity-pools/blob/main/audits/2023-09-SRLabs.pdf)                          |
+| Spearbit  | [`October 2023 - Cantina Managed Report`](https://github.com/centrifuge/liquidity-pools/blob/main/audits/2023-10-Spearbit-Cantina-Managed.pdf) |
 
 ## License
+
 This codebase is licensed under [GNU Lesser General Public License v3.0](https://github.com/centrifuge/liquidity-pools/blob/main/LICENSE).
+
+```
```
Comment (hieronx) [inline README.md:120]: ?

File: README.md [hunk 5]
```diff
@@ -7,39 +8,115 @@
...
 
 #### Testing
+
 To run all tests locally:
+
 ```sh
 forge test
 ```
 
+## Deploying
```
Comment (hieronx) [inline README.md:44]: Thanks for adding docs!!

**Review by hieronx — COMMENTED** [review]
I'll try to do a test deployment later still, but generally looks great to me!

**Comment by AStox** [comment]
Closed because Sphinx has shut down and is no longer being actively developed/supported.

**Pull request closed without merging**